kinder/ci/workflows/skew-1.20-on-1.19.yaml

```diff
@@ -1,12 +1,12 @@
 version: 1
```

Comment:
hm, the diff here says that this is a rename `skew-1.19-on-1.18.yaml → kinder/ci/workflows/skew-1.20-on-1.19.yaml`

we still need the `skew-1.19-on-1.18.yaml` file

Reply:
included file skew-1.19-on-1.18.yaml